addons/website_sale/models/website_snippet_filter.py

```diff
@@ -163,11 +163,14 @@ def _get_products_latest_viewed(self, website, limit, domain, **kwargs):
                 ).search(domain, limit=limit)
         return products
 
+    def _get_products_and_template(self, template_id):
+        template_id = template_id and self.env['product.template'].browse(int(template_id)).exists()
+        return self.env['product.product'], template_id
+
```

Comment:
Really not worth making a new method for that.

Reply:
Ok, I will revert this. I will use it in method directly. I have made a method just for don't repeat code in another.